pkg/model/api/config/frontend.go

```diff
@@ -60,4 +80,6 @@ type Frontend struct {
 	ImportantDashboards []dashboardSelector `json:"important_dashboards,omitempty" yaml:"important_dashboards,omitempty"`
 	// TimeRange contains the time range configuration for the dropdown
 	TimeRange TimeRange `json:"time_range,omitempty" yaml:"time_range,omitempty"`
+	// BannerInfo contains the content to be display in a banner at the top of each page along with the severity of the information
+	BannerInfo Banner `json:"banner,omitempty" yaml:"banner,omitempty"`
```

Comment:
```suggestion
BannerInfo *Banner `json:"banner,omitempty" yaml:"banner,omitempty"`
```
Having a pointer makes sense here as the banner is not something we need by default

Besides it will allow to enforce the Verify method for the `Banner`